metis-common/metis-common-zoho/src/main/java/eu/europeana/metis/zoho/ZohoUtils.java

```diff
@@ -22,8 +26,28 @@ private ZohoUtils() {
    */
   public static String stringFieldSupplier(Object object) {
     if (!JSONObject.NULL.equals(object)) {
-      return object.toString();
+      if (object instanceof Choice<?>) {
+        return ((Choice<?>) object).getValue().toString();
+      } else {
+        return object.toString();
+      }
     }
     return null;
   }
+
+  /**
+   * Method that would check if the object provided is of type {@link JSONObject.Null} and will
+   * return a correct representation of {@link List} with {@link String} items or empty list.
+   *
+   * @param object the object to be checked
+   * @return the List of strings representation of the object or empty list
+   */
+  public static List<String> stringListSupplier(Object object) {
+    if (!JSONObject.NULL.equals(object) && object instanceof List<?> && ((List<?>) object)
+        .get(0) instanceof Choice<?>) {
```

Comment:
Can this list be empty? Should we check that?

Reply:
Yes it's possible, I've added a check